planner/core/logical_plans.go

```diff
@@ -1169,3 +1169,37 @@ type LogicalShowDDLJobs struct {
 
 	JobNumber int64
 }
+
+// CTEClass holds the information and plan for a CTE.
+type CTEClass struct {
+	IsDistinct               bool
+	seedPartLogicalPlan      LogicalPlan
+	recursivePartLogicalPlan LogicalPlan
+	cteTask                  task
```

Comment:
Which physical plan does it represent? Why `PhysicalCTE` also holds two physical plan?

Reply:
cteTask is the physical plan for the whole CTE. PhysicalCTE has two physical plans for the seed part and the recursive part.

Reply:
We could comment the relation between them to help others understand the codes.